content/developer/reference/javascript/services.rst

```diff
@@ -475,3 +481,174 @@ A notification that closes after a second
   const notificationService = useService("notification");
   const close = notificationService.add("I'll will be quickly closed");
   setTimeout(close, 1000);
+
+.. _services/effect:
+
+Effect service
+--------------
+
+Overview
+~~~~~~~~
+
+* Technical name: `effect`
+* Dependencies: None
+
+Effects are graphical elements that can be temporarily displayed on top of the page, usually to provide feedback to the user that something interesting happened.
+
+A good example would be the rainbow man:
+
+.. image:: images/rainbow_man.png
+    :alt: The rainbow man effect
+    :width: 600
+    :align: center
+
+
+Here's how this can be displayed:
+
+.. code-block:: javascript
+
+    const effectService = useService("effect");
+    effectService.add({
+      type: "rainbow_man",
+      message: "Boom! Team record for the past 30 days.",
+    });
+
+.. warning :: 
+    The hook `useEffect` is not related to the effect service.
+
+API
+~~~
+
+.. js:function:: add(options)
```

Comment:
Red runbot is caused by this function because function  `add` is specified twice on the page (and in the doc).

`WARNING: duplicate function description of add, other function in developer/reference/javascript/services`

You have to link it to a specific class to differentiate the two methods.

```rst
.. js:class:: EffectService()

  .. js:method: add(params)

```

(Like it's done by Gery in #1245)